server/mlx_runner.py

```
@@ -0,0 +1,1050 @@
+        For MXFP4 models that generate reasoning followed by final answer,
+        format it nicely for display.
+        """
+        if not self._is_reasoning_model:
+            return response
+            
+        # Check if response contains reasoning markers
+        if self._reasoning_start in response and self._final_start in response:
+            # Extract reasoning and final parts
+            try:
+                # Split on the reasoning start
+                before_reasoning, after_start = response.split(self._reasoning_start, 1)
```

Ruff (0.14.1)

Unpacked variable `before_reasoning` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)